components/inputs/input-time.js

```diff
@@ -403,15 +412,10 @@ class InputTime extends SkeletonMixin(FormElementMixin(LitElement)) {
 	}
 
 	async _handleKeydown(e) {
-		const dropdown = this.shadowRoot.querySelector('d2l-dropdown-menu');
 		// open and focus dropdown on down arrow or enter
 		if (e.keyCode === 40 || e.keyCode === 13) {
-			if (!this._dropdownFirstOpened) {
-				this._dropdownFirstOpened = true;
-				await this.updateComplete;
-			}
-			dropdown.open(true);
```

Comment:
When opening, `applyFocus` is true by default so this behaviour should remain the same. The `this.shadowRoot.querySelector('d2l-menu').focus();` also seems to happen by default on key open